test/plugins/typings.test.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env jest */
+
+const network = require('../../src/lib/network');
+const typingsPlugin = require('../../src/plugins/typings');
+const { success, warning } = require('../../src/lib/format');
+
+jest.mock('../../src/lib/network');
+jest.mock('../../src/lib/format', () => ({
+  ...jest.requireActual('../../src/lib/format'),
+  failure: jest.fn(),
+  success: jest.fn(),
+  warning: jest.fn()
+}));
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+it('should return null if the package specifies typings info on package.json', async () => {
```

Comment:
Maybe this could be
```suggestion
it('should return null if the package specifies types info on package.json', async () => {
```
so that it's different from the description of the following test?

Reply:
Looks great! I left the description the same because I thought `meh why not?` but now I can see the confusion.